editor/selectors.js

```diff
@@ -19,7 +19,7 @@ import { addQueryArgs } from '@wordpress/url';
  * @return {String}       Editing mode
  */
 export function getEditorMode( state ) {
-	return state.mode;
+	return state.preferences.mode || 'visual';
```

Comment:
Should we use `getPreference( state 'mode' )` here? Is defaulting something which we might consider building into the `getPreference` selector (maybe a third argument)?